controllers/apps/v2alpha1/generated.go

```diff
@@ -167,7 +179,7 @@ func generateListenerService(instance *appsv2alpha1.EMQX, listenerPorts []corev1
 			Namespace:   instance.Namespace,
 			Name:        instance.Spec.ListenersServiceTemplate.Name,
 			Labels:      instance.Spec.ListenersServiceTemplate.Labels,
-			Annotations: instance.Spec.ListenersServiceTemplate.Annotations,
+			Annotations: mergeMap(instance.Spec.ListenersServiceTemplate.Annotations, annotations),
```

Comment:
I think this merge should in default web hook

Reply:
> I think this merge should in default web hook

ok, I agree with you